.devcontainer/Dockerfile

```diff
@@ -1 +1,71 @@
-FROM composablefi/composable-devcontainer:latest
+ARG USER=vscode
+ARG UID=1000
+ARG GID=${UID}
+ARG NIX_INSTALLER=https://releases.nixos.org/nix/nix-2.10.3/install
+ARG CHANNEL_URL=https://github.com/NixOS/nixpkgs/archive/aaa1c973c8c189195e1b1a702d3b74dbcde91538.tar.gz
+ARG CACHIX_NAME=composable-community
+FROM mcr.microsoft.com/vscode/devcontainers/base:0.202.7-bullseye
+
+SHELL [ "/bin/bash", "-o", "pipefail", "-o", "errexit", "-c" ]
+
+ARG CHANNEL_URL
+ARG USER
+ARG UID
+ARG GID
+ARG NIX_INSTALLER
+ARG CACHIX_NAME
```

Comment:
are they duplicates?

Reply:
i do no know. as far as I know it for sure allows to set default for docker file, but if you inherit you are forced to set your defaults (cause these are not set).

Reply:
so it to make only from with defaults, then inherits will get defaults. as I see people do not do that and i will retain.